tests/Assetic/Test/Asset/HttpAssetTest.php

```diff
@@ -15,14 +15,14 @@
 
 class HttpAssetTest extends \PHPUnit_Framework_TestCase
 {
-    const JQUERY = 'http://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js';
```

Comment:
I don't understand. I see mentioned error on failed build: https://travis-ci.org/kriswallsmith/assetic/jobs/203755333 , but probably it was failing before this PR as well.